fix: fix primary owner group notifications not sent

ytvnr opened the pull request:
## Issue

https://gravitee.atlassian.net/browse/APIM-1871

## Description

When looking for a primary owner group, build it with the email of the primary owner member

<!-- Storybook placeholder -->
---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-upuopccwpm.chromatic.com)
<!-- Storybook placeholder end -->

ytvnr committed:
fix: fix primary owner group notifications not sent
https://gravitee.atlassian.net/browse/APIM-1871
https://github.com/gravitee-io/issues/issues/9079

jgiovaresco commented on gravitee-apim-rest-api/gravitee-apim-rest-api-model/src/main/java/io/gravitee/rest/api/model/PrimaryOwnerEntity.java:
```diff
@@ -44,9 +44,16 @@ public PrimaryOwnerEntity(UserEntity user) {
     }
 
-    public PrimaryOwnerEntity(GroupEntity group) {
+    //    public PrimaryOwnerEntity(GroupEntity group) {
```
Do we need to keep this comment 🤔

jgiovaresco commented on gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/GroupService.java:
```diff
@@ -22,6 +22,7 @@
 import io.gravitee.rest.api.service.common.ExecutionContext;
 import java.util.List;
 import java.util.Set;
+import org.jetbrains.annotations.NotNull;
```
It seems unused

jgiovaresco commented on gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/GroupServiceImpl.java:
```diff
@@ -69,7 +69,6 @@
 import java.util.*;
 import java.util.function.Consumer;
 import java.util.stream.Collectors;
-import org.checkerframework.checker.units.qual.C;
```
It seems unused

jgiovaresco reviewed (approved):
I think we should not commit the `package-lock.json` changes in this PR to help mergify 🙈

ytvnr committed:
chore: fix conflict in package-lock.json
https://gravitee.atlassian.net/browse/APIM-1871
https://github.com/gravitee-io/issues/issues/9079
